lib/pbench/cli/agent/commands/results/__init__.py

```diff
@@ -58,42 +56,42 @@ def move_results(ctx, _user, _prefix, _show_server):
         logger.error("Failed to create temporary directory")
         sys.exit(1)
 
-    dirs = [
-        _dir
-        for _dir in next(os.walk(config.pbench_run))[1]
-        if not _dir.startswith("tools-") and not _dir.startswith("tmp")
-    ]
+    runs_copied = 0
+    failures = 0
```

Comment:
Are you even using `failures` anymore? You dropped the `crt.copy_result_tb()` return, which used to trigger incrementing the value. Now we're just ignoring failure to copy?

Reply:
Yes, the test uses `failures`, the main code that calls this method is broken in many other ways, but I was not going to try to fix that brokenness as well.

And we don't ignore copy failures.  All of this code calls `sys.exit()`, which is a HUGE mistake, and really my mistake because I code reviewed this and failed to flag this as an issue.  So rather than try to fix all that, I was trying to focus on just fixing the test to actually take action.

The return value from `crt.copy_result_tb` is always None.  So you could never run a successful test here.

And the return value from `mrt.make_result_tb` could only be a `string`, which means the `if result_tb_name` was always `True`.

We'll have to tackle the `sys.exit()` issue, which is what is leading to this mess in another PR.

Reply:
I can file an issue for better handling `sys.exit()`  to track if you have not filed yet

Reply:
filed and issue https://github.com/distributed-system-analysis/pbench/issues/1955